lib/ansible/modules/web_infrastructure/sophos_utm/utm_dns_host.py

```diff
@@ -0,0 +1,155 @@
+        default: False
+        type: bool
+    timeout:
+        description:
+          - the timeout for the utm to resolve the ip address for the hostname again
+        default: 0
+
+extends_documentation_fragment:
+    - utm
+"""
+
+EXAMPLES = """
+- name: Create UTM dns host entry
+  utm_dns_host:
+    utm_host: sophos.host.name
+    utm_token: abcdefghijklmno1234
+    name: TestDNSEntry
+    hostname: testentry.some.tld
+    state: present
+
+- name: Remove UTM dns host entry
+  utm_dns_host:
+    utm_host: sophos.host.name
+    utm_token: abcdefghijklmno1234
+    name: TestDNSEntry
+    state: absent
+"""
+
+RETURN = """
+result:
+    description: The utm object that was created
+    returned: success
+    type: complex
+    contains:
+        _ref:
+            description: The reference name of the object
+            type: string
+        _locked:
+            description: Whether or not the object is currently locked
+            type: boolean
+        name:
+            description: The name of the object
+            type: string
+        address:
+            description: The ipv4 address of the object
+            type: string
+        address6:
+            description: The ipv6 adress of the object
+            type: string
+        comment:
+            description: The comment string
+            type: string
+        hostname:
+            description: The hostname of the object
+            type: string
+        interface:
+            description: The reference name of the interface the object is associated with
+            type: string
+        resolved:
+            description: Whether the ipv4 address is resolved or not
+            type: boolean
+        resolved6:
+            description: Whether the ipv6 address is resolved or not
+            type: boolean
+        timeout:
+            description: The timeout until a new resolving will be attempted
+            type: int
+"""
+
+from ansible.module_utils.utm_utils import UTM, UTMModule
+
+
+def main():
+    endpoint = "network/dns_host"
+    key_to_check_for_changes = ["comment", "hostname", "interface"]
+    module = UTMModule(
+        argument_spec=dict(
+            name=dict(type='str', required=True),
+            address=dict(type='str', required=False, default='0.0.0.0'),
+            address6=dict(type='str', required=False, default='::'),
+            comment=dict(type='str', required=False, default=""),
+            hostname=dict(type='str', required=False),
+            interface=dict(type='str', required=False, default=""),
+            resolved=dict(type='bool', required=False, default=False),
+            resolved6=dict(type='bool', required=False, default=False),
+            timeout=dict(type='int', required=False, default=0),
+        )
+    )
+    try:
+        UTM(module, endpoint, key_to_check_for_changes).execute()
+    except Exception as e:
+        module.fail_json(msg=str(e))
```

Comment:
Can you replace `str` with `to_native` ? 

```python
from ansible.module_utils._text import to_native

...
module.fail_json(msg=to_native(e))
```

Reply:
implemented